nix/overlay.nix

```diff
@@ -7,7 +7,11 @@ self: super: {
     ];
   }));
 
-  pythonForAnsible = (self.python3.withPackages (_: self.ansible.requiredPythonModules));
+  pythonForAnsible = (self.python3.withPackages (_: self.ansible.requiredPythonModules ++ [
```

Comment:
This won't do anything afaik.    

`self.ansible.requiredPythonModules` is defined as     `filter hasPythonModule drv.propagatedBuildInputs` 
https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/python-packages.nix#L80

so `requiredPythonModules` should already have those three packages in there.